encryption/ntru-prime/lprime/bench/optimized_ntrulpr653_xeon-e3-1220-skylake.yaml

```diff
@@ -0,0 +1,10 @@
+platform: xeon-e3-1220-skylake
+impl: factored
```

Comment:
There is a mismatch between the filename and this attribute.

Reply:
If that's something we rely on, it should be checked by the validator script.

Reply:
PR #23 addresses that.